sui-execution/latest/sui-adapter/src/programmable_transactions/execution.rs

```diff
@@ -371,9 +398,19 @@ mod checked {
                             This should be a coin"
                         );
                     };
+                    input_infos.push((balance.value(), *id.object_id()));
```

Comment:
It's probably fine to collect these outside of tracing, but just to be safe/consistent I'd probably guard this in a similar fashion to `add_move_value_info_from_value` and have it only push these values in if tracing is set.

Reply:
I figured it's just a clone of already obtained value, but it certainly would not hurt to do it consistently with the other trace helpers, so done!